parsers/fortran.c

```diff
@@ -582,10 +582,11 @@ static void makeLabelTag (vString *const label)
 
 static lineType getLineType (void)
 {
-	vString *label = vStringNew ();
+	static vString *label = NULL;
```

Comment:
@b4n, are you talking about "reachable"? 
Consider a memory object assigned to a static pointer variable. As far as NULL (or something other memoryu object) is assigned to the variable, I don't say it is "leaked". Instead "it is still reachable at the end of process.". These two are different things as valgrind reported.